src/cloud-code/httpRequest.js

```diff
@@ -1,25 +1,36 @@
 var request = require("request"),
   Parse = require('parse/node').Parse;
 
-var encodeBody = function(body, headers = {}) {
+var encodeBody = function(options = {}) {
```

Comment:
Do you mean that you need to `inout` modify it or you can get away with just returning the body+headers from this function?